### DISCUSSION TITLE
an official extension to support csharp-ls

### DISCUSSION THREAD

**Discussion by amirongit** [neoclide/coc.nvim#4932] [Extension request]
I really like coc.nvim and the simplicity it offers; but there is not official & maintained extension to support csharp-ls.
since https://github.com/coc-extensions/coc-omnisharp is archived, too, would it be possible to have an extension which integrates the csharp-ls lsp and do the decompilation please? (since the integration cannot be fully done using the "languageservers" property of coc config)
more information: https://github.com/neoclide/coc.nvim/discussions/4917

**Comment by amirongit** [comment]
I'm adding a comment to keep this request alive 😅

**Comment by yaegassy** [comment]
@amirongit I've created a coc.nvim extension for csharp-ls. https://github.com/yaegassy/coc-csharp-ls
